package.json

```diff
@@ -148,6 +154,11 @@
           "type": "boolean",
           "default": true,
           "description": "Enable/Disable hover over on module name"
+        },
+        "ansible.deploymentTemplatesGitHubRepo": {
```

Comment:
will local source be supported later?

Reply:
possibly, but i think it's not necessary now. if anyone requests we can do this.